.changeset/wet-lemons-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cloudflare/pages-shared": patch
+"wrangler": patch
+---
+
+Less verbose warnings from header and redirect parsing
```

Comment:
```suggestion
fix: reduce verbose logging from header and redirect parsing in `wrangler pages dev`

Previously information and warning errors were always logged to the console even if the log level was set to ERROR.
```

Reply:
This message isn't quite correct - the issue was that these were always logged as warnings, when most of them are info messages

Reply:
Isn't that more or less what I suggested?

Reply:
What about:
`fix: utilise logging levels correctly in createMetadataObject, and reuse the original Logger type instead of redefining a subset`
Let me know your thoughts and I can commit whichever is the most suitable.

Reply:
Sorry to churn on this @magnusdahlstrand - the changesets are used to build the changelog, so these messages should be written in the context of a user understanding what has changed. We don't need to go into the detail of the implementation just what is different for the user. So no need to mention the `Logger` type or even `createMetadataObject` as these are internal.

How about?

```suggestion
fix: Use appropriate logging levels when parsing headers and redirects in `wrangler pages dev`.
```